datajunction-server/datajunction_server/api/attributes.py

```diff
@@ -29,8 +29,13 @@ def list_attributes(*, session: Session = Depends(get_session)) -> List[Attribut
     return session.exec(select(AttributeType)).all()
 
 
-@router.post("/attributes/", response_model=AttributeType, status_code=201)
-def add_an_attribute_type(
+@router.post(
+    "/attributes/",
+    response_model=AttributeType,
+    status_code=201,
+    name="Add An Attribute Type",
```

Comment:
I think this looks better when we don't capitalize the articles `a` or `an`.

Reply:
@agorajek I noticed that FastAPI does that for us anyway, but I can change it here in case we figure out how to disable that at some point 😢